ts/components/TosWebviewComponent.tsx

```diff
@@ -108,12 +104,12 @@ const TosWebviewComponent: React.FunctionComponent<Props> = ({
     renderError()
   ) : (
     <>
-      <View style={{ flex: 1 }} testID={"toSWebViewContainer"}>
+      <View style={IOStyles.flex} testID="toSWebViewContainer">
         <WebView
           androidCameraAccessDisabled={true}
           androidMicrophoneAccessDisabled={true}
           textZoom={100}
-          style={{ flex: 1 }}
+          style={IOStyles.flex}
           onLoadEnd={handleLoadEnd}
           onError={handleError}
```

Comment:
I suggest to provide also `onHttpError` to check all sort of network errors. What do you think?

```suggestion
          onError={handleError}
          onHttpError={handleError}
```